services/app-api/src/emailer/emails/newPackageCMSEmail.test.ts

```diff
@@ -750,8 +752,10 @@ test('does not include state specific analyst on contract and rate submission',
     })
 })
 
-test('includes oactEmails on contract and rate submission', async () => {
```

Comment:
Can we also add the test here that you did for unlock - oact doesn't get the emails if its a non risk based contract.